src/components/color-picker/color-picker.tsx

```diff
@@ -0,0 +1,75 @@
+import { Component, h, State } from '@stencil/core';
+
+/**
+ * This component enables you to select a swatch from out color palette, simply
+ * by clicking on it. You can then copy the css variable name of the chosen color
+ * and use it where desired.
+ *
+ * The color picker can also show you a preview of any valid color name or color value.
+ *
+ * @exampleComponent limel-example-color-picker
+ */
+
+@Component({
+    tag: 'limel-color-picker',
+    shadow: true,
+    styleUrl: 'color-picker.scss',
+})
+export class ColorPicker {
+    /**
+     * Name or code of the chosen color
+     */
+    @State()
+    public color: string;
+
+    @State()
+    private isOpen = false;
+
+    public render() {
+        const background = this.color ? { '--background': this.color } : {};
+
+        return (
+            <div class="color-picker">
+                <limel-popover open={this.isOpen} onClose={this.onPopoverClose}>
+                    <div
+                        class="picker-trigger"
+                        slot="trigger"
+                        style={background}
+                        role="button"
+                        tabindex="0"
+                        onClick={this.openPopover}
+                        title="Click to pick a color"
+                    />
+                    <limel-color-picker-palette
+                        searchedColor={this.color}
+                        onChange={this.onChosenColor}
+                    />
+                </limel-popover>
+
+                <limel-input-field
+                    label="Chosen color"
+                    helperText="You can also type a color name or value to preview it here"
```

Comment:
We should not hard code these, they should probably be props on the component iteself. For full compatibility with the `Form` component, we should implement as many as the props from the `FormComponent` interface as possible, otherwise we might have to create a wrapper component if we need any other of the other ones when we want to use it in a form

Reply:
> Also, after clicking on a color in the picker I think it would be great if we could highlight that color somehow so I can see what color I clicked on 🙂

That's right. We had the same idea. Should we do it in this PR or later? We thought we could add it later.

Reply:
> We should not hard code these, they should probably be props on the component iteself. For full compatibility with the Form component, we should implement as many as the props from the FormComponent interface as possible, otherwise we might have to create a wrapper component if we need any other of the other ones when we want to use it in a form

Oh we forgot to put them in translations files (there is also a duplicate of them, one in the palette component and one in the picker. I think the text is not synced neither if I remember correctly). Do you mean we should instead make them string props, and let them rather be translated when implemented by consumer?

Reply:
> That's right. We had the same idea. Should we do it in this PR or later? We thought we could add it later.

I don't want to merge the component in this state. If you mark it as private then you can do whatever :smile:

Reply:
> Do you mean we should instead make them string props, and let them rather be translated when implemented by consumer?

I don't think all of these should be in the translation files. The consumer should be able to control some of these labels, just as a label for a regular input field. E.g. in Lime Admin I imagine that we want to change the label to something like "Limetype color" and the description to something like "Pick a color that will be used for this Limetype in icons and headers"

Reply:
"Also, after clicking on a color in the picker I think it would be great if we could highlight that color somehow so I can see what color I clicked on "

@jgroth Yes, I will commit this changes , just fixed it

Reply:
OK. Sorry. It's my bad. I moved it back to "in progress" again to fix these things first. ;)